src/Orchard.Web/Modules/Orchard.Autoroute/Drivers/AutoroutePartDriver.cs

```diff
@@ -144,6 +144,10 @@ public class AutoroutePartDriver : ContentPartDriver<AutoroutePart> {
                 if (!_autorouteService.IsPathValid(part.DisplayAlias)) {
                     updater.AddModelError("CurrentUrl", T("Please do not use any of the following characters in your permalink: \":\", \"?\", \"#\", \"[\", \"]\", \"@\", \"!\", \"$\", \"&\", \"'\", \"(\", \")\", \"*\", \"+\", \",\", \";\", \"=\", \", \"<\", \">\", \"\\\", \"|\", \"%\", \".\". No spaces are allowed (please use dashes or underscores instead)."));
                 }
+                
+                if (part.DisplayAlias != null && part.DisplayAlias.Length > 1850){
+                    updater.AddModelError("CurrentUrl", T("Your Display Alias is too long. The Display Alias can only be up to 1,850 characters."));

```

Comment:
Please use the word "Permalink" instead of "Display Alias" to be consistent with the other validation messages.